test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -69,10 +69,10 @@ public bool CheckEquality()
                 return _testCmd.CheckEquality(_dir, _name);
             }
 
-            public bool CheckEqualityNormalized()
+            public bool CheckEqualityNormalized(decimal precision = 10000000)
```

Comment:
Can we define this const somewhere, so we know that everywhere is using the same number and not off by one or two zeros?

Reply:
Nit, not worth respinning the PR, but you could write as `10_000_000` to be easier to read.